File: Data Structures & Algorithms/buy-and-sell-crypto/submission-0.java
class Solution {
    public int maxProfit(int[] prices) {
        int max = 0;
        int curr = 0;
        int future = 0;

        for(int i =0; i<prices.length; i++)
        {
            for(int j = i+1; j<prices.length; j++)
            {
                curr = prices[i];
                future = prices[j];
                if(prices[j]-prices[i] > max)
                {
                    max = prices[j]-prices[i];
                }
            }
        }
        return max;
    }
}

File: Data Structures & Algorithms/largest-unique-number/submission-0.py
class Solution:
    def largestUniqueNumber(self, nums: List[int]) -> int:
        freq = {}
        for num in nums:
            freq[num] = freq.get(num, 0) + 1
        maxval = -1
        for key, val in freq.items():
            curr = key
            if val == 1:
                maxval = max(curr, maxval)
        return maxval
            


        